themes/bootstrap3/js/vendor/autocomplete.js

```diff
@@ -276,6 +276,16 @@ function Autocomplete(_settings) {
           },
           false
         );
+        window.addEventListener(
```

Comment:
autocomplete.js is a separate project (note that this is in a vendor directory), so changes to this code should be made through a PR to https://github.com/vufind-org/autocomplete.js. It looks to me like this might be a general improvement not directly tied to sticky elements in any case.

Reply:
Sorry, I missed that this is in vendor. I created a PR in the project https://github.com/vufind-org/autocomplete.js/pull/15. Should I remove this here or can it stay since it will be included anyway?

Reply:
It's fine to leave it for now; I've added a TODO checkbox to the top of this PR to be sure we don't merge this one before the autocomplete updates are done.